src/uwtools/config.py

```diff
@@ -70,6 +72,15 @@ def __init__(self, config_path=None):
 
         self.config_path = config_path
 
+        #initialize logger with parent inheritance
+        py_caller = getmodule(stack()[1][0])
+        if py_caller.__name__ ==  __name__:
+            name = __name__
+        else:
+            name = f"{py_caller.__name__}.{__name__}"
+        log = logging.getLogger(name)
```

Comment:
Should be `self.log` so we can use it throughout.

Maybe also `_name`. (Per Brian in person)